### DISCUSSION TITLE
[El-date-picker] on change input return invalid date

### DISCUSSION THREAD

**Discussion by yuniit** [element-plus/element-plus#17378] [Q&A]
Any help is appreciated

https://github.com/element-plus/element-plus/assets/30621352/7b502cd2-4b83-4838-8ae6-4a8210178339

[Playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6Ijx0ZW1wbGF0ZT5cbiAgPGRpdiBjbGFzcz1cImJsb2NrXCI+XG4gICAgPGRpdj5cbiAgICAgIHt7IHZhbHVlMSB9fVxuICAgIDwvZGl2PlxuXG4gICAgPGVsLWRhdGUtcGlja2VyXG4gICAgIHYtbW9kZWw9XCJ2YWx1ZTFcIiB0eXBlPVwiZGF0ZXRpbWVyYW5nZVwiXG4gICAgIHJhbmdlLXNlcGFyYXRvcj1cIlRvXCIgXG4gICAgIHN0YXJ0LXBsYWNlaG9sZGVyPVwiU3RhcnQgZGF0ZVwiXG4gICAgICBlbmQtcGxhY2Vob2xkZXI9XCJFbmQgZGF0ZVwiIFxuICAgIHN0eWxlPVwibWFyZ2luLXRvcDogMTZweDtcIiAvPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWYsd2F0Y2ggfSBmcm9tICd2dWUnXG5cbmNvbnN0IHZhbHVlMSA9IHJlZjxbRGF0ZSwgRGF0ZV0+KFtcbiAgbmV3IERhdGUoMjAyNCwgNSwgMTAsIDAsIDApLFxuICBuZXcgRGF0ZSgyMDI0LCA1LCAxNSwgMCwgMCksXG5dKVxuXG53YXRjaCh2YWx1ZTEsKCk9PntcbiAgY29uc29sZS5sb2coJ3ZhbHVlMScsdmFsdWUxLnZhbHVlWzBdKVxufSlcblxuPC9zY3JpcHQ+XG48c3R5bGUgc2NvcGVkPlxuLmJsb2NrIHtcbiAgcGFkZGluZzogMzBweCAwO1xuICB0ZXh0LWFsaWduOiBjZW50ZXI7XG4gIGJvcmRlci1yaWdodDogc29saWQgMXB4IHZhcigtLWVsLWJvcmRlci1jb2xvcik7XG4gIGZsZXg6IDE7XG59XG5cbi5ibG9jazpsYXN0LWNoaWxkIHtcbiAgYm9yZGVyLXJpZ2h0OiBub25lO1xufVxuXG4uYmxvY2sgLmRlbW9uc3RyYXRpb24ge1xuICBkaXNwbGF5OiBibG9jaztcbiAgY29sb3I6IHZhcigtLWVsLXRleHQtY29sb3Itc2Vjb25kYXJ5KTtcbiAgZm9udC1zaXplOiAxNHB4O1xuICBtYXJnaW4tYm90dG9tOiAyMHB4O1xufVxuPC9zdHlsZT5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IFwiRVNOZXh0XCIsXG4gICAgXCJqc3hcIjogXCJwcmVzZXJ2ZVwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwidHlwZXNcIjogW1wiZWxlbWVudC1wbHVzL2dsb2JhbC5kLnRzXCJdLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZSxcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsIl9vIjp7fX0=)

**Comment by warmthsea** [comment]
You can use `value-format="YYYY-MM-DD"`

![screenshots](https://github.com/element-plus/element-plus/assets/45450994/1379d08d-fff3-442a-96ce-1973d4a8e7b9)
